src/static/javascripts/components/dashboard.jsx

```diff
@@ -121,7 +121,7 @@ var MainDiv = React.createClass({
 		if (!loggedIn()) {
 		   return (<Welcome />);
 		}
-        var itemNodes = testQuestions.map(function (item) {
+        var itemNodes = data.map(function (item) {
```

Comment:
@Bradyta you're correct. `{% raw survey_json %}` gives the raw data that should be delivered when you first visit the page (think facebook feed. It remembers you're latest posts and recent content, but then there is a seperate live mechanism for updating the data).

Updating the data "live" with the ajax requests is what you're working on, right?